sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1-basic-operations.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief This sample provides the code implementation to use the Key Vault Secrets SDK client for
+ * C++ to create, get, update, delete and purge a secret.
+ *
+ * @remark The following environment variables must be set before running the sample.
+ * - AZURE_KEYVAULT_URL:  To the Key Vault account URL.
+ * - AZURE_TENANT_ID:     Tenant ID for the Azure account.
+ * - AZURE_CLIENT_ID:     The Client ID to authenticate the request.
+ * - AZURE_CLIENT_SECRET: The client secret.
+ *
+ */
+
+#if defined(_MSC_VER)
+#define _CRT_SECURE_NO_WARNINGS
+#endif
+
+#include <azure/core.hpp>
```

Comment:
don't think we need this

Reply:
done and cleaned up all the other unused includes in all the other samples